### ISSUE TITLE
Reported grading error on challenge

### ISSUE THREAD

**Issue by weaverryan** [knpuniversity/oo-challenges#2]
From UserVoice:

knpuniversity.com/screencast/oo-ep3/broken-ship/activity/291

> printing any value below 3000 results in failing challenge (Hmm, did you print the DeathStarIII description in h3 tag?) when this value isn't indicated in instructions to challenge

@bocharsky-bw can you look at this? Let me know what you find, and I can update the user :).

**Comment by bocharsky-bw** [comment]
Yes, I tweak the grading in https://github.com/knpuniversity/oo-challenges/commit/ed51c5648b404bc2840acb9801e4dbc72ee4f1f6 . Now user can return any number :)

I'm wondering how to deploy changes to prod?

**Comment by weaverryan** [comment]
@bocharsky-bw awesome! What was the significance of 3000 - is that some value that is in the class by default? Right now, to deploy, when you edit the oo-ep3 screencast in the admin area, there is a textbox at the bottom called `Challenges repository sha` - update that to the new sha on this repository and it will update :). I'll let you do make that change :).

Thanks!

**Comment by bocharsky-bw** [comment]
`3000` was our hardcoded value at solution step, but I missed that fact that user can enter another value. That's why my grading failed.
